qa/tasks/kafka.py

```diff
@@ -116,13 +117,17 @@ def run_kafka(ctx,config):
                 ],
             )
 
+            time.sleep(90)
```

Comment:
maybe instead of sleep, run something like `pgrep kafka` in a loop, to make sure that the process is down?
same thing for the zookeeper.
(you would need to put some limit on that loop)

if the processes hangs, you can `kill -9 <pid>` if you figure out their pid, or just `killall java` (hoping there are no other java processes running on that machine...)